src/Pelagos/Bundle/AppBundle/Resources/views/layout.html.twig

```diff
@@ -95,20 +95,51 @@
                 </div>
             </div>
             {% include "PelagosAppBundle::layout-menu.html.twig" %}
-            <div id="pelagos-container">
-                <div id="pelagos-container-row">
-                    <div id="pelagos-container-cell">
-                        <div id="pelagos-wrapper">
-                            <div id="pelagos-wrapper-row">
-                                <div id="pelagos-content">
-                                    <h1 style="margin: 0 auto;">{% block pagetitle %}{% endblock %}</h1>
-                                    {% block body %}{% endblock %}
+            <div id="main-wrapper"><div id="main">
+
+                    <div id="content-wrapper">
+                        <div id="content-main" class="column">
+                            <div class="section">
+                                <div id="outer-container">
+                                    <div>
```

Comment:
What's this stray div doing here?

Reply:
Just wanted to show you the original drupal's div. we only need one extra div

Reply:
It works tho